### ISSUE TITLE
Sega Dreamcast PCB

### ISSUE THREAD

**Issue by pmgducati** [pmgducati/Blue-Retro-AIO-Units#18]
I do not have this system this PCB is compatible with, please put your findings here.

**Comment by joeybaumgartner** [comment]
I'm doing a build of this and hope to fully test this soon. Not sure if this is due to routing or not, but the P1 through P4 solder points are reversed compared to all of the other 4 player units I've assembled so far (N64 and GC). This just makes it so that the cables running out the 3D printed case cross over each other and harder to tighten up. I've just run the wires out straight and made a note to myself that P4 = P1, P3 = P2, P2 = P3, and P1 = P4.  Not sure if it's easer to update the PCB, or make an alternate box to 3D print.

**Comment by pmgducati** [comment]
Hey,
Thanks for the heads up, I must have missed that when making the PCB.  I have made the changes and the pads now match the others and housing.
![image](https://user-images.githubusercontent.com/20894227/189570622-5bb6c864-ef65-4e4e-b13a-60d47ea067d9.png)

**Comment by pmgducati** [comment]
Keep me updated on how your project goes!

**Comment by corim123** [comment]
I'm working on some of the GC adapters(well waiting on DB25s) and plan on getting some of these boards next as I've got a Dreamcast kicking around somewhere.

**Comment by joeybaumgartner** [comment]
Sorry for such a late reply on this. Still working with the original boards (ordered them before noticing the issue). I have had some success where I can connect a controller up to the Dreamcast, but cannot get it to recognize the Start button on either an XBox One controller or with a Wii-mote connected to a SNES Classic controller. 

@corim123 I have had no issues with the GC adapters so far, and I used the Wii-mote + SNES Classic controller for playing GBA games via GB Player and that's 1000 times better than using a GC controller.

**Comment by pmgducati** [comment]
I just picked up a dreamcast and ordered the PCBs myself.  I will update when the PCBs arrive in a couple weeks

**Comment by joeybaumgartner** [comment]
That sounds great! I'm interested to hear your results.

**Comment by pmgducati** [comment]
@joeybaumgartner Finally got the PCBs assembled and my Dreamcast setup and I am not having any issues with my setup.  I have tested with 3 different 8bitdo controllers (SN30 Pro+, Pro 2 and N30), 2x PS4 controllers and 2x Xbox One controllers.  I played for about 4 hours with different combinations of all my controllers and single and multiplayer.  I tested Sonic adventure, zero gunner 2, MVC2, DOA 2, Mr Driller, Hydro Thunder and Daytona USA.

I am going to close this comment as I am not seeing any issues with so many games.  I you are still have issues please open a new issue.
![PXL_20221128_165207633](https://user-images.githubusercontent.com/20894227/204386522-c8da730b-cd55-478c-9e4a-ea86e13f36f2.jpg)
![PXL_20221128_212921694](https://user-images.githubusercontent.com/20894227/204386527-73fea0ea-842c-4d85-ad90-50635119b4c6.jpg)
![PXL_20221128_172338586](https://user-images.githubusercontent.com/20894227/204386531-9fd84cab-9d86-47e8-ae9a-e062c55a96ad.jpg)
